wallet/test/Test/Pos/Wallet/Web/Methods/BackupDefaultAddressesSpec.hs

```diff
@@ -0,0 +1,87 @@
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
+{-# OPTIONS_GHC -fno-warn-unused-matches #-}
+{-# OPTIONS_GHC -fno-warn-unused-local-binds #-}
+module Test.Pos.Wallet.Web.Methods.BackupDefaultAddressesSpec
+       ( spec
+       ) where
+
+import           Universum
+
+import           Data.Default (def)
+import           Formatting ()
+import           Pos.Crypto (firstHardened)
+import           Pos.Crypto.Signing.Types.Safe (emptyPassphrase)
+import           Pos.Launcher (HasConfigurations)
+import           Pos.Util.BackupPhrase (mkBackupPhrase9, bpToList)
+import           Pos.Util.CompileInfo (HasCompileInfo, withCompileInfo)
+import           Pos.Wallet.Web.Account (GenSeed(..))
+import           Pos.Wallet.Web.Backup (WalletBackup(..), getWalletBackup)
+import           Pos.Wallet.Web.ClientTypes (CWallet(..), CWalletMeta(..),
+                                             CWalletInit(..), CWalletAssurance(..),
+                                             Wal(..), CId(..), CHash(..), encToCId)
+import           Pos.Wallet.Web.Methods.Backup (restoreWalletFromBackup)
+import           qualified Pos.Wallet.Web.Methods.Logic as L
+import           Pos.Wallet.Web.Methods.Restore (newWallet)
+import           Pos.Wallet.Web.State (AddressLookupMode(Ever), getAccountWAddresses,
+                                       getWalletMeta, getWalletAddresses)
+import           Pos.Wallet.Web.Util (getWalletAccountIds)
+import           Test.Hspec (Spec, describe)
+import           Test.Pos.Util (assertProperty, stopProperty, expectedOne,
+                                withDefConfigurations)
+
+import Test.Pos.Wallet.Web.Mode (walletPropertySpec)
+
+
+spec :: Spec
+spec = withCompileInfo def $
+       withDefConfigurations $
+       describe "restoreAddressFromWalletBackup" $ restoreWalletAddressFromBackupSpec
+
+restoreWalletAddressFromBackupSpec :: (HasCompileInfo, HasConfigurations) => Spec
+restoreWalletAddressFromBackupSpec = walletPropertySpec restoreWalletAddressFromBackupDesc $ do
+    -- I NEED FIRST TO CREATE WALLETBACKUP -> HOW ?
```

Comment:
@gromakovsky I left some code comments so you can see what is the problem for me.